data.json

```diff
@@ -593,6 +593,15 @@
     "dateAdded": "2019-10-04T16:13:00.000Z",
     "tags": ["hacktoberfest", "stickers", "accessories", "expired"]
   },
+  {
+    "name": "Vonage",
+    "difficulty": "medium",
+    "description": "First, sign up on the Hacktoberfest site. To qualify for the official limited edition Hacktoberfest shirt, you must register and make four pull requests between October 1-31. Pull requests can be to any public repo on GitHub, not just the ones with issues labeled Hacktoberfest. If a maintainer reports your pull request as spam or spammy behavior, or not in line with the project’s code of conduct, you will be ineligible to participate.",
```

Comment:
Please make it a bit more light, and precise what repos we're talking about.

Reply:
Also precise the kind of swag, from the "What swag can I earn from Vonage?" section in their FAQ.

Reply:
Thank you for your comments. The description is updated